ingest/downloader/flattener.py

```diff
@@ -23,11 +27,13 @@ def _flatten_entity(self, entity, object_key):
 
         if not object_key:
             content = entity['content']
-            worksheet_name = self._get_concrete_entity(content)
+            concrete_entity = self._get_concrete_entity(content)
```

Comment:
Do we need the `concrete_entity` variable?

Reply:
Ah, i was gonna reuse it in the _extract_schema_url but decided not to. I'll remove it. Thanks!